src/main/java/uk/gov/cshr/service/InviteService.java

```diff
@@ -60,6 +71,7 @@ public void sendEmail(Invite invite) throws NotificationClientException {
         NotificationClient client = new NotificationClient(api);
         SendEmailResponse response = client.sendEmail(templateId, invite.getForEmail(), personalisation, "");
 
+        // TODO: 25/04/2018 Matt - remove this log later, just using for dev purposes to output email contents so we can get the signup activation link
```

Comment:
DOD - Left debug code

Reply:
This was intentional for now as we are still using the test api key. When we cut over, ill take it out. But until then I don't want to forget about it

Reply:
OK